packages/dashboard/src/components/side-panel/shared/expandableSectionHeader.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC } from 'react';
+import { Icon } from '@cloudscape-design/components';
+import './styles.css';
+
+const ExpandableSectionHeader: FC<{ text: string; onClickButton?: (e: Event) => void }> = ({ text, onClickButton }) => {
+  return (
+    <>
+      {text}
+      {onClickButton && (
+        <span className="expandable-section-header-icon">
+          <Icon name="add-plus" />
```

Comment:
Good opportunity to take the icon name in as a prop but provide a default.

Reply:
agree